### DISCUSSION TITLE
튜토리얼 기능 수동화 제안

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#202] [Ideas]
메뉴 버튼에 도움말, 혹은 튜토리얼 버튼을 만들어서, 유저가 원할 경우 튜토리얼을 확인할 수 있도록 하는 기능을 제안합니다.
참고할 만한 자료는 프로그래머스 기본 조작방법 튜토리얼 입니다.

아래 사진 첨부합니다.
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/22653737/c44eadff-ca0b-4801-9890-12a4742a89cc)
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/22653737/377182d8-cabd-49e0-9d11-24d5f902478e)
![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/22653737/0a301ea3-5d03-431a-8b52-908a72dc6a30)

**Comment by cstrnull00** [comment]
그럼 저희 지금 작성된 튜토리얼하고는 관계 없나요?

**Reply by mille-feu** [reply]
현재 작성된 튜토리얼과 무관합니다.

현재 논의중이었던 튜토리얼을 1안이라고 가정하고, 이 기능을 2안으로써 제안드립니다.
튜토리얼의 의의는 남겨두되, 그 기능과 역할을 축소시킨 느낌으로 봐주시면 되겠습니다.

**Comment by seong0929** [comment]
- 1안 대체하여 아래 방식은 어떠할까요?
  - ![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/446a3df7-7075-47ae-8463-88b026c470d2)
  - ![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/a183c4fb-1004-42a7-abe9-5fbfc382dc75)
  - ![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/5e42be10-4b84-4b95-98e6-308df1a4c15c)
  - ![image](https://github.com/Tiny-Metro/Tiny-Metro/assets/50050003/7919e13b-e138-4277-85cc-c29ffe814929)
  - ...
